pkg/images/defaults/flavor.go

```diff
@@ -190,6 +195,38 @@ func RHACSReleaseImageFlavor() ImageFlavor {
 	}
 }
 
+// OpenSourceReleaseImageFlavor returns image values for `opensource` flavor.
+func OpenSourceReleaseImageFlavor() ImageFlavor {
+	v := version.GetAllVersionsDevelopment()
+	return ImageFlavor{
+		MainRegistry:       "quay.io/stackrox-io",
+		MainImageName:      "main",
+		MainImageTag:       v.MainVersion,
+		CentralDBImageTag:  v.MainVersion,
+		CentralDBImageName: "central-db",
+
+		CollectorRegistry:      "quay.io/stackrox-io",
+		CollectorImageName:     "collector",
+		CollectorImageTag:      v.CollectorVersion + "-latest",
+		CollectorSlimImageName: "collector",
+		CollectorSlimImageTag:  v.CollectorVersion + "-slim",
+
+		ScannerImageName:       "scanner",
+		ScannerSlimImageName:   "scanner-slim",
+		ScannerImageTag:        v.ScannerVersion,
+		ScannerDBImageName:     "scanner-db",
+		ScannerDBSlimImageName: "scanner-db-slim",
+
+		ChartRepo: ChartRepo{
+			URL: "https://charts.stackrox.io",
```

Comment:
This isn't going to work because, I believe, https://charts.stackrox.io requires authentication. Open Source users will not be able to download charts from there.
Maybe there's a way to use the future https://github.com/stackrox/helm-charts/opensource here? Please add a TODO to address this thing.
```suggestion
			// TODO(ROX-10079): establish a place where open source users will be able to download Helm charts.
			URL: "https://charts.stackrox.io",
```

Reply:
True! In discussions, we settled with putting the charts URL there for now. I agree that adding a TODO here and updating this as soon as ROX-10079 is done is the way to go. I've updated said ticket to ensure we update the flavor once it is getting implemented.